src/common/components/Input/Icons/ViewOn.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+export const ViewOn = (
```

Comment:
I was thinking we could just put this in a ViewOn.svg file and just keep the `<svg>...</svg>` part.